src/comm/bench-flow.js

```diff
@@ -221,37 +217,11 @@ async function defaultTest(args, clientArgs, final) {
             trim: args.trim ? args.trim : 0,
             args: args.arguments,
             cb  : args.callback,
-            config: configPath,
+            config: configPath
         };
         // condition for time based or number based test driving
         if (args.txNumber) {
             msg.numb = testRounds[i];
-            // File information for reading or writing transaction request
-            msg.txFile = {roundLength: testRounds.length, roundCurrent: i, txMode: args.txMode};
-            if(args.txMode && args.txMode.type === 'file-write') {
-                logger.info('------ Prepare(file-write) waiting ------');
-                msg.txFile.readWrite = 'write';
-                msg.rateControl = {type: 'fixed-rate', opts: {tps: 400}};
-                try {
-                    await client.startTest(msg, clientArgs, function(){}, testLabel);
-                    msg.numb = testRounds[i];
-                    msg.txFile.readWrite = 'read';
-                    msg.rateControl = args.rateControl[i] ? args.rateControl[i] : {type:'fixed-rate', 'opts' : {'tps': 1}};
-                    if(i === (testRounds.length - 1)) {
-                        logger.info('Waiting 5 seconds...');
-                        logger.info('------ Prepare(file-write) success------');
-                        await Util.sleep(5000);
-                    }
-                } catch (err) {
-                    logger.error('------Prepare(file-write) failed------');
-                    args.txMode.type = 'file-no';
-                }
-
-            }else if(args.txMode && args.txMode.type === 'file-read'){
-                msg.txFile.readWrite = 'read';
-            }else {
-                msg.txFile.readWrite = 'no';
-            }
```

Comment:
The test flow is deleted, so how can caliper complete the test? I'm confused.

Reply:
See previous comment.